spacialgaze-plugins/misc.js

```diff
@@ -433,4 +435,26 @@ exports.commands = {
 		Db.disabledScrolls.remove(target);
 	},
 	enableintroscrollhelp: ["/enableintroscroll [room] - Enables scroll bar preset in the room's roomintro."],
+
+	'!ai': true,
+	askai: 'ai',
+	'helix': 'ai',
+	'qmark': 'ai',
+	'8ball': 'ai',
+	ai: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		target = (target.replace(/^[^a-z0-9]+/i, "")).trim();
+		const query = encodeURI(target);
```

Comment:
This needs to be `encodeURIComponent`, if I'm not mistaken. Also, you need to check if `target` is still a valid string (not an empty one) after all leading non-alphanumeric characters are replaced.

On second thought, I think the regex needs to be `/^[^A-Za-z0-9]+/i` so that it doesn't strip away leading capital letters.